.travis.yml

```diff
@@ -1,3 +1,23 @@
 language: java
 jdk:
- - oraclejdk8
\ No newline at end of file
+ - oraclejdk8
+script: mvn clean install --batch-mode
+matrix:
+ - jdk: oraclejdk8
+branches:
+ only:
+  - 2.1.x
+after_success:
+ - mvn jacoco:report coveralls:report
+notifications:
+ email: false
+ irc:
+  channels:
+   - "irc.freenode.org#openmrs"
+  template:
+   - "%{repository}#%{build_number} (%{branch} - %{commit} : %{author}): %{message}"
+   - "Change view : %{compare_url}"
+   - "Build details : %{build_url}"
+  use_notice: true
+  skip_join: true
+sudo: false
```

Comment:
Can you give a quick summary of what this does?  I"m not particularly familiar with Travis configuration.